apps/meteor/app/api/server/v1/teams.ts

```diff
@@ -82,10 +82,11 @@ API.v1.addRoute(
 					members: Match.Maybe([String]),
 					room: Match.Maybe(Match.Any),
 					owner: Match.Maybe(String),
+					sidepanel: Match.Maybe(Match.Any),
```

Comment:
We're trying to avoid using `check` for new params since we'll remove it from all endpoints on 7.0. Can you validate this param using an if statement instead? It'd be great if we could check if it contains valid values in the items field